{"issue_type": "request_error", "error_msg": "HTTPConnectionPool(host='doe.state.wy.us', port=80): Max retries exceeded with url: /lmi (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f0d23375a90>, 'Connection to doe.state.wy.us timed out. (connect timeout=15)'))"}